generate/vim_template/themes/hackerman/hackerman.vim

```diff
@@ -0,0 +1,241 @@
+" URL: https://github.com/whatyouhide/vim-hackerman
```

Comment:
@CorbinPost10 I think you don't need to copy this file here. 

This file can be simple as: 
```vim
colorscheme hackerman
```